lib/frizz/local.rb

```diff
@@ -32,6 +33,15 @@ def expand_path(local_path)
       ::File.join root_path, local_path
     end
 
+    def ignore_for_gzip_version?(local_path)
+      return false unless @options[:prefer_gzip]
+
+      return false if local_path.ends_with? '.gz'
+
+      gzip_file_version_path = "#{expand_path(local_path)}.gz"
+      return true if ::File.file?(gzip_file_version_path)
+    end
+
     def ignore?(path)
       ::File.directory?(path) || ignorance.ignore?(path)
```

Comment:
Actually, this should probably be refactored out into `ignorance` as that's the object currently responsible for deciding if a file should be ignored for the project